docker/complement/conf-workers/postgres.supervisord.conf

```
@@ -0,0 +1,11 @@
+[program:postgres]
+command=/usr/bin/pg_ctlcluster 13 main start --foreground
```

it might be worth using a script to prefix log lines. something like:

`/usr/local/bin/prefix-log`:

```sh
#!/bin/bash

exec 1> >(awk '{print "'"${SUPERVISOR_PROCESS_NAME}"' | "$0}' >&1)
exec 2> >(awk '{print "'"${SUPERVISOR_PROCESS_NAME}"' | "$0}' >&2)
exec "$@"
```

and then:

```suggestion
command=/usr/local/bin/prefix-log /usr/bin/pg_ctlcluster 13 main start --foreground
```